lib/wraft_doc_web/router.ex

```diff
@@ -104,6 +104,14 @@ defmodule WraftDocWeb.Router do
       # Set Password for first time
       post("/users/set_password", UserController, :set_password)
     end
+
+    # templates
+    scope "/v1", Api.V1, as: :v1 do
+      scope "/templates" do
+        get("/", TemplateAssetController, :list_public_templates)
+        get("/:file_name/download", TemplateAssetController, :download_public_template)
+      end
+    end
```

Comment:
Keep the scope as template_assets so that consistency is maintained with the rest of the APIs in the template_asset controller